overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -12430,6 +12490,24 @@
             }
           },
           "taskID:8": "bq_standard:fluid"
+        },
```

Comment:
Combine into one task.

Reply:
Not possible because one is fluid and one is item, and we don't want to use the bucket form.